test/integration/parse-param-operator-in-operator/01-assert.yaml

```diff
@@ -25,4 +25,4 @@ spec:
       containers:
         - env:
           - name: PARAM_ENV
-            value: "after"
\ No newline at end of file
+            value: "after"
```

Comment:
what is the change here?

Reply:
It's the missing newline being added @kensipe - look at the panel on the left.

![Zrzut ekranu z 2020-03-03 09-54-19](https://user-images.githubusercontent.com/489420/75758899-f8ad2200-5d34-11ea-9f72-1c15ee329552.png)